qemu/tests/timerdevice_boot.py

```diff
@@ -134,13 +135,25 @@ def update_clksrc(session, clksrc):
     timerdevice_host_load_cmd = params.get("timerdevice_host_load_cmd")
     if timerdevice_host_load_cmd:
         error_context.context("Add some load on host", logging.info)
-        process.system(timerdevice_host_load_cmd, shell=True,
-                       ignore_bg_processes=True)
+        host_cpu_cnt_cmd = params["host_cpu_cnt_cmd"]
+        host_cpu_cnt = int(process.system_output(host_cpu_cnt_cmd, shell=True).strip())
+        if params["os_type"] == "linux":
+            timerdevice_host_load_cmd = timerdevice_host_load_cmd % host_cpu_cnt
+            process.system(timerdevice_host_load_cmd, shell=True,
+                           ignore_bg_processes=True)
+        else:
+            timerdevice_host_load_cmd = timerdevice_host_load_cmd % int(host_cpu_cnt/2)
+            stress_bg = utils_test.HostStress("stress", params,
+                                              stress_args=timerdevice_host_load_cmd)
+            stress_bg.load_stress_tool()
```

Comment:
`load_stress_tools` is used to run stress test in vm, not host.

Reply:
Pls see line 146, the class is HostStress. So the stress is run on host.

Reply:
Not exactly, even if it is 'utils_test.HostStress', please check the function itself, thanks!

Reply:
Hi yanan, I have checked the function before using it, no matter the function itself and the test results, I think the stress can be used on host.

Reply:
You are right,  it is my mistake that i missed the session_handler, thanks!